packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -19,9 +19,9 @@ import { IContainerRuntimeMetadata, ICreateContainerMetadata } from "../summary"
 export type GCVersion = number;
 
 /** The stable version of garbage collection in production. */
-export const stableGCVersion: GCVersion = 1;
+export const stableGCVersion: GCVersion = 2;
```

Comment:
I wonder... should we leave stable as 1? I think if 1-2 succeeds then the system is proven and subsequent bumps can be done freely. If we haven't completed that roll-out when this code gets deployed, I think we'd want to be able to roll back to 1, not to 2.